src/clj/rems/handler.clj

```diff
@@ -22,7 +22,9 @@
 (defn public-routes []
   (routes
    (home/home-routes)
-   (auth/auth-routes)))
+   ;; never cache authentication results
+   ;; TODO this is a slightly hacky place to do this
+   (middleware/wrap-no-cache (auth/auth-routes))))
```

Comment:
I wonder if we should actually default to `Cache-Control: no-store` in our middleware and explicitly set `Cache-Control: max-age=23h` for static resources

Reply:
Possibly that would be the safest.